### PR TITLE
initial improvement for more resources

### PR DESCRIPTION
**Pull request by AlaShibanAtKlo**
### Standard checks

- **Unit tests**: Any special considerations?
- **Docs**: Do we need to update any docs, internal or public?
- **Backwards compatibility**: Will this break existing apps? If so, what would be the extra work required to keep them working?

### CODE REVIEW

File: pkg/templates/aws/resources/s3_bucket_policy.yaml [hunk 1]
```diff
@@ -37,7 +39,7 @@ properties:
                 type: map(string,string)
               StringLike:
                 type: map(string,string)
-              Null:
+              null:
```
Comment (jhsinger-klotho) [inline pkg/templates/aws/resources/s3_bucket_policy.yaml:42]: we definitely dont want the actual property names changed, was this intended?

File: pkg/templates/aws/resources/iam_role.yaml [hunk 2]
```diff
@@ -73,7 +79,7 @@ properties:
                     type: map(string,string)
                   StringLike:
                     type: map(string,string)
-                  Null:
```
Comment (jhsinger-klotho) [inline pkg/templates/aws/resources/iam_role.yaml:76]: same here

File: pkg/templates/aws/resources/iam_role.yaml [hunk 3]
```diff
@@ -33,7 +36,7 @@ properties:
                 type: map(string,string)
               StringLike:
                 type: map(string,string)
-              Null:
+              null:
```
Comment (jhsinger-klotho) [inline pkg/templates/aws/resources/iam_role.yaml:39]: and here

**Comment by gordon-klotho** [comment]
Is this PR still relevant?

**Comment by gordon-klotho** [comment]
Closing per discussion. Keeping the branch in case we bring it back

**Pull request closed without merging**